XWebView.podspec

```diff
@@ -90,7 +90,7 @@ Pod::Spec.new do |s|
   #
 
   s.source_files  = "XWebView/*.swift"
-  s.exclude_files = "XWebView/XWVInvocation.swift"
+  s.exclude_files = "XWebView/XWVInvocation.swift", "XWebView/XWVHttp*.swift"
```

Comment:
I'm getting a warning from XCode:
`XWVInvocation is missing from module XWebView.framework`